script/crypto-assets-importer/tron/trc10-tokens.js

```diff
@@ -1,5 +1,5 @@
 var whitelist = [
-  1002000, // BTT
+  // 1002000, // BTT
```

Comment:
Is that comment expected, shouldn't it be removed?
(Genuine question, I have no idea how it's supposed to work. :smile: )

Reply:
I'll remove it